examples/Dockerfile.libfabric

```diff
@@ -143,9 +143,16 @@ RUN git clone https://github.com/openpmix/openpmix.git \
  && rm -Rf ../openpmix*
 
 # FLUX-PMI
-ARG FLUX_VERSION=0.45.0
+#
 # Flux requires a number of additional packages. We install them here to
 # distinquish between libfabric provider and flux-pmi dependencies.
+#
+# Flux pmi library, libpmi.so, conflicts with SLURM pmi2. To use FLUX you must
+# set LD_PRELOAD, as follows, in the container at run time.
+#
+#ENV LD_PRELOAD=/usr/local/lib/flux/libpmi.so
+#
```

Comment:
In principle we could change the name of one or the other, or configure `/usr/local/lib/flux to be a higher priority than wherever Slurm’s library is.

Reply:
Hmm, good point, I'll try that.

Reply:
Hmm, I think the issue here was actually due to misconfiguring Openmpi by linking the wrong flux-pmi library. See new commit f493eab. I was incorrectly linking `-lflux-core`, instead of `-lpmi` as prescribed by the `flux-pmi.pc` file.